packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/sections/components/InvoicePreview.tsx

```
@@ -364,41 +316,85 @@
-                      </td>
-                      <td>{formatTaxRate(item)}%</td>
-                    </tr>
-                  ))
+                  sanitizedItems.map((item) => {
+                    const taxRate = getRateValueForItem(item);
+                    const totalAmount =
+                      (item?.product?.price?.value || 0) *
+                      (item?.quantity || 1);
+
+                    return (
+                      <tr>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)